app/ledger.js

```diff
@@ -2093,6 +2093,19 @@ var networkConnected = underscore.debounce(() => {
  * low-level utilities
  */
 
+var muonWriter = (path, payload) => {
+  muon.file.writeImportant(path, JSON.stringify(payload, null, 2), (success) => {
+    if (!success) return console.log('write error: ' + path)
+
+    if ((quitP) && (!getSetting(settings.PAYMENTS_ENABLED)) && (getSetting(settings.SHUTDOWN_CLEAR_HISTORY))) {
```

Comment:
i'm not sure whether it makes sense. it is certainly counter-intuitive: if you have successfully written the file, and the user is quitting, and the ledger isn't enabled, and history is to be deleted, then get rid of the synopsis file... reminiscent of https://www.youtube.com/watch?v=GoxcyV4DJuY